### ISSUE TITLE
Possible copyrighted image in the Beam docs

### ISSUE THREAD

**Issue by moorepants** [sympy/sympy#24388]
I just noticed this image:

![image](https://user-images.githubusercontent.com/276007/207676282-f0729afc-b154-4cec-84dd-936e179566f1.png)

from:

https://docs.sympy.org/latest/modules/physics/continuum_mechanics/beam.html#sympy.physics.continuum_mechanics.beam.Beam.plot_ild_moment

It looks like a typical image from a text book and thus could possibly be a copyright violation in our docs.

The PR is https://github.com/sympy/sympy/pull/21810 and there seems to be no discussion over the use of the image.

The second issue is that it is a bitmap image instead of vector (e.g. svg).

This image should likely be removed unless the copyright can be verified.

cc @ishanaj @Psycho-Pirate

**Comment by moorepants** [comment]
Here is a second one:
![image](https://user-images.githubusercontent.com/276007/207678887-b35617b5-0325-48f3-a9ff-c51d2e236959.png)

**Comment by 1e9abhi1e10** [comment]
can I make these diagrams by my own ?

**Comment by Psycho-Pirate** [comment]
I wasn't aware of the possible copyright violation while making that pull request. Maybe we can use the draw function to create an image for these examples.

**Comment by 1e9abhi1e10** [comment]
> > can I make these diagrams by my own and add into the docs ?
> 
> Yes, that is the best solution. Please make SVG images if you do.

Ok, I will.

**Comment by 1e9abhi1e10** [comment]
![WhatsApp Image 2022-12-16 at 19 34 35](https://user-images.githubusercontent.com/95904102/208115818-f0948c93-70e0-4048-bcc4-409d955ce546.jpeg)
Is it fine ?

**Comment by moorepants** [comment]
Looks good. Can you make the two supports clearly a pin on the left and a sphere (point contact) on the right?

**Comment by moorepants** [comment]
The black arrow on the top should align with the black dot between A and B also.

**Comment by moorepants** [comment]
> The black arrow on the top should align with the black dot between A and B also.

Sorry, I was incorrect about this. You have it right.

**Comment by moorepants** [comment]
cm -> m

**Comment by 1e9abhi1e10** [comment]
![4 m](https://user-images.githubusercontent.com/95904102/208174789-26fbeb3a-388f-4e3d-a3fd-4e117387cbf9.jpeg)
Now it's fine?

**Comment by 1e9abhi1e10** [comment]
![WhatsApp Image 2023-01-03 at 19 35 00 (1)_page-0001](https://user-images.githubusercontent.com/95904102/210381500-aaa26469-b1d0-49aa-b5f3-ecbd421c9072.jpg)

**Comment by 1e9abhi1e10** [comment]
![as_page-0001](https://user-images.githubusercontent.com/95904102/210387331-b5b1b2f1-9153-4dfd-bc5b-7a535915ffe3.jpg)

**Comment by asTejaswinis** [comment]
Should the aspect ratios of the images be kept same?

I want to upload the following images
![ildreaction](https://user-images.githubusercontent.com/95971605/217616464-d899f17f-8e07-4f0a-90c2-73789f81af98.png)

![Beam2new](https://user-images.githubusercontent.com/95971605/217616720-381d48b0-8b33-4891-a778-49808eb16c9b.PNG)

Are they good to go?

Their aspect ratios are different from current images

**Comment by moorepants** [comment]
If we are going to replace this, we should replace it with an SVG image that is editable. We shouldn't add bitmaps to the repo, as a general rule.

**Comment by asmeurer** [comment]
We should just remove the image from the docs if we don't own the copyright to it. We can work on getting a replacement later. @Psycho-Pirate where did you get this image from?

**Comment by asmeurer** [comment]
There are several png files in the repo:

```py
$ git ls-files | grep png
doc/src/_static/sympylogo.png
doc/src/_static/sympylogo_big.png
doc/src/modules/physics/continuum_mechanics/allowed-sign-conventions.png
doc/src/modules/physics/continuum_mechanics/ildreaction.png
doc/src/modules/physics/continuum_mechanics/ildshear.png
doc/src/modules/physics/continuum_mechanics/truss_example.png
doc/src/modules/physics/mechanics/examples/multidof-holonomic.png
doc/src/pics/consoleascii.png
doc/src/pics/consoleunicode.png
doc/src/pics/ipythonnotebook.png
doc/src/pics/ipythonqtconsole.png
doc/src/pics/pngview1.png
sympy/plotting/tests/test_region_and.png
sympy/plotting/tests/test_region_not.png
sympy/plotting/tests/test_region_or.png
sympy/plotting/tests/test_region_xor.png
```

Presumably all the ones in doc/src/modules/physics should be removed and replaced with SVGs. `doc/src/modules/physics/continuum_mechanics/ildreaction.png` looks like it comes from the same source as `doc/src/modules/physics/continuum_mechanics/ildshear.png`, the one shown in the screenshot above.  The rest appear to be hand-made, but we should confirm that they are. 

The other png files are screenshots or other things which are not a problem.

**Comment by asmeurer** [comment]
I've made a PR removing the questionable images at https://github.com/sympy/sympy/pull/24691. Like I said, we should still review the other images, and make SVGs for them otherwise. We can also make a replacement for these images, but they need to be removed regardless if we do not own the copyright for them.

**Comment by asmeurer** [comment]
@Psycho-Pirate in the future, do not add anything to SymPy (images, code, or otherwise) unless you yourself explicitly created it. If it's something that someone else created, you need to make sure you have explicit permission, and include full attribution. 

SymPy's docs were already taken down before due to a false copyright claim (https://www.asmeurer.com/blog/posts/the-sympy-hackerrank-dmca-incident/). If we have things in the repo that we actually don't own the copyright to, it could result to the docs and the entire repo being taken down. 

And in general, do not add binary files like PNGs to the repo. There should be a very good reason to do so if you do. Images like this in the docs should be SVGs so that they take up less space in the repo and can be edited.

**Comment by asTejaswinis** [comment]
> If we are going to replace this, we should replace it with an SVG image that is editable. We shouldn't add bitmaps to the repo, as a general rule.

can't i replace the existing ildreaction.png file and other png file with the ones i shared above

**Comment by asmeurer** [comment]
@asTejaswinis did you construct those images yourself (including the image of the beam)? If so, you should be able to produce a vector version of the image from whatever program you used to construct it. If you did not, then we cannot include it, for the same reason we removed the original, unless you have explicit permission from whoever did produce them.

**Comment by asmeurer** [comment]
By the way, the aspect ratios do not matter. The images are only used as figures.

**Comment by asTejaswinis** [comment]
> @asTejaswinis did you construct those images yourself (including the image of the beam)? If so, you should be able to produce a vector version of the image from whatever program you used to construct it. If you did not, then we cannot include it, for the same reason we removed the original, unless you have explicit permission from whoever did produce them.

Yes, I have created the figures in 3d modeling software(Fusion 360) and labeled the snips of them using another software(simple photo editor). I will check if they can produce svg files
Nevertheless, I am able to delete those original png files and just add these files as .png. I want to know if I can proceed with it and pull a request

**Comment by asmeurer** [comment]
The original PNG files have already been removed in #24691. I don't think we should add new figures unless they are vector images that we have the ability to edit in the future.
